### ISSUE TITLE
Has Check

### ISSUE THREAD

**Issue by khrome83** [xpyxel/dash#25]
**Is your feature request related to a problem? Please describe.**
Currently, you have to `get()` the item which increases the amount of code to run, when you just need to know if it exists. 

**Describe the solution you'd like**
Expose the `.has()` check map already has.

**Additional context**
In a perfect world, it would make sense to have -

```ts
if (!cache.has(key)) {
  // get item from data
  cache.set(key, data);
  return data;
} else {
  return cache.get(key);
}
```

**Comment by xpyxel** [comment]
Merged #24 to patch this, thanks.

**Comment by khrome83** [comment]
TY for the quick response!
